specification/billing/resource-manager/Microsoft.Billing/preview/2018-03-01-preview/billing.json

```diff
@@ -199,8 +199,8 @@
           }
         }
       }
-    },
-    "/subscriptions/{subscriptionId}/providers/Microsoft.Billing/invoices": {
+    },   
```

Comment:
we are moving our new API "discoverTenants" from "Microsoft.Billing" to "Microsoft.Consumption". All the change we made previously remains the same except for different folder and path.